.github/workflows/test-build-docker.yml

```diff
@@ -86,13 +94,345 @@ jobs:
           aws s3 cp s3://${{ secrets.S3_INTEGRATION_BUCKET }}/${{ inputs.BucketKey }}/linux/amd64/amazon-cloudwatch-agent.deb amd64/amazon-cloudwatch-agent.deb
           aws s3 cp s3://${{ secrets.S3_INTEGRATION_BUCKET }}/${{ inputs.BucketKey }}/linux/arm64/amazon-cloudwatch-agent.deb arm64/amazon-cloudwatch-agent.deb
 
-      - name: Build Cloudwatch Agent Image
+      - name: Get ECR Repo name
+        id: repo_name
+        env:
+          ContainerRepositoryNameAndTag: ${{ inputs.ContainerRepositoryNameAndTag }}
+        run: |
+          RepoName=`echo $ContainerRepositoryNameAndTag | awk -F: '{print $1}'`
+          echo "::set-output name=ContainerRepositoryName::$RepoName"
+
+      - name: Build Cloudwatch Agent Image amd64
```

Comment:
Why did we separate the image build for amd64 and arm64?

Reply:
When `docker/build-push-action@v4` pushes images for both `amd64` and `arm64 ` together. Manifest list was getting uploading as well. Manifest list if not created 
I separated these out so that individual image is uploaded and at the end we create Manifest list with both Linux and Windows images.